### PR TITLE
[RUM-20850][RUM-20935][DOCS] prettify examples and improve readme 

### PR DESCRIPTION
**Pull request by yghern**

### CODE REVIEW

File: examples/Vanilla/Conversion/index.html [hunk 1]
```diff
@@ -36,21 +36,17 @@ <h1>Conversion</h1>
...
-
-
-    <!--
-    Make sure that you run one of the following 2 commands:
-    webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/Conversion/conversion.js  --scope implementation --rulesTarget legacy --mode development
-    webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/Conversion/conversion.js  --scope implementation --rulesTarget current --mode development
-
-    and then uncomment the appropriate one
+    <!--  Make sure that you run one of the following 2 commands and then uncomment the appropriate one below
+        webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/CustomCollector/customCollector.js  --scope implementation --rulesTarget legacy --mode development
```
Comment (robertdumitrescu) [inline examples/Vanilla/Conversion/index.html]: shouldn't the commands point to Conversion directory?

File: examples/Vanilla/Error/index.html [hunk 2]
```diff
@@ -9,22 +9,21 @@
...
-    <!--    <script src="https://sdk.real-user-data.eggplant.cloud/latest/rci.min.js"></script>-->
-
-    <!--
-    Make sure that you run one of the following 2 commands:
-    webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/Error/error.js  --scope implementation --rulesTarget legacy --mode development
-    webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/Error/error.js  --scope implementation --rulesTarget current --mode development
-
-    and then uncomment the appropriate one
+    <!--  Make sure that you run one of the following 2 commands and then uncomment the appropriate one below
+        webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/CustomCollector/customCollector.js  --scope implementation --rulesTarget legacy --mode development
```
Comment (robertdumitrescu) [inline examples/Vanilla/Error/index.html]: Same here

File: examples/Vanilla/index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```
Comment (robertdumitrescu) [inline examples/Vanilla/index.html:1]: Awesome addition 👍

Comment (robertdumitrescu) [reply examples/Vanilla/index.html:1]: Shouldn't this also include onload implementation?

Comment (yghern) [reply examples/Vanilla/index.html:1]: it have onload (line 16)

File: examples/Vanilla/Conversion/index.html (hunk 1 above)
Comment (yghern) [reply examples/Vanilla/Conversion/index.html]: ok fixed

File: examples/Vanilla/Error/index.html (hunk 2 above)
Comment (yghern) [reply examples/Vanilla/Error/index.html]: ok fixed

File: examples/Vanilla/index.html (hunk 3 above)
Comment (yghern) [reply examples/Vanilla/index.html:1]: OK updated

**Comment by robertdumitrescu** [comment]
Code review OK from my perspective

**Review by robertdumitrescu — APPROVED** [review]

**Review by MalcolmSmithUK — APPROVED** [review]

**Pull request merged**